charts/payment-api/values.preview.template.yaml

```diff
@@ -22,5 +22,17 @@ java:
     #Below key required for preview pipeline
     GOV_PAY_AUTH_KEY_CMC: 8ln764q6jrbgqvvvv40uggsea3h1p602v79gdrt7ni66mrmpk416ulohvd
     LAUNCH_DARKLY_SDK_KEY: sdk-06456c8b-312c-475c-b154-ca585f5ca063
+    PCI_PAL_ANTENNA_TENANT_NAME: HMCTS
+    PCI_PAL_ANTENNA_USER_NAME: APIUser
+    PCI_PAL_ANTENNA_CLIENT_ID: HMCTSStage
+    PCI_PAL_ANTENNA_CLIENT_SECRET: Vm82kOUZAf9U4Kms3hIQM03jqsDQKc8aHIGy5I9y
+    PCI_PAL_ANTENNA_GET_TOKENS_URL: https://pcipalstaging.cloud/api/v1/token
+    PCI_PAL_ANTENNA_LAUNCH_URL: https://euwest1.pcipalstaging.cloud/api/v1/session/303/launch
+    PCI_PAL_ANTENNA_VIEW_ID_URL: https://euwest1.pcipalstaging.cloud/session/303/view/
```

Comment:
I think we are hardcoding TenentId [303] as part of the urls, it would be better if we make this configurable like keeping the TenentId in App Properties and build the urls dynamically

Reply:
It won't change frequently.  Just we need to change it in 2 places. If we need to have this configurable we need to create new vaults for tenantId and change the vault value of existing one's as well in all the environments.